### PR TITLE
Change the version navigation to make it clearer

### PR DESCRIPTION
**Pull request by NolwennP**
It will now look like this.
Screenshot 2020-04-27 at 18 07 23
Screenshot 2020-04-27 at 18 08 18

### CODE REVIEW

**Comment by LaureBro** [comment]
nice!!

File: src/partials/version-navigation.handlebars [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="nav navbar-nav navbar-right">
+    <li class="active">
+        <a href="/pim/serenity/index.html">Serenity</a>
+    </li>
+    <li id="last-EE-version" data-filepath="{{filePath}}" data-major-version="{{majorVersion}}"></li>
+    <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Other versions <span class="caret"></span></a>
+        <ul class="dropdown-menu" id="version-dropdown" data-filepath="{{filePath}}" data-major-version="{{majorVersion}}">
```
Comment (NolwennP) [inline src/partials/version-navigation.handlebars:8]:
```suggestion
        <ul class="dropdown-menu" id="version-dropdown" data-filepath="{{filePath}}" data-current-doc-version="{{majorVersion}}">
```

File: src/js/versions.js [hunk 2]
```diff
@@ -20,15 +20,29 @@ const updateVersions = function() {
...
+                            versionsDom.classList.add("active");
+                        }
+                    } else {
+                        versionsDom = document.getElementById('version-dropdown');
+                        filepath = versionsDom.dataset.filepath;
+                        link.href = version.suffix_with_filename ? version.url + filepath : version.url;
+                        var node = document.createElement('li');
+                        node.appendChild(link);
+                        versionsDom.appendChild(node);
+                        if(version.version === versionsDom.dataset.majorVersion){
```
[the remark this answers is not in the document]
Comment (ahocquard) [reply src/js/versions.js:42]: :+1:  and in line 32 also

File: src/js/versions.js [hunk 3]
```diff
@@ -20,15 +20,29 @@ const updateVersions = function() {
...
-                    link.href = version.suffix_with_filename ? version.url + filepath : version.url;
                     link.innerHTML = version.label;
-                    node.appendChild(link);
-                    versionsDom.appendChild(node);
+                    if(key === 0) {
+                        versionsDom = document.getElementById('last-EE-version');
+                        filepath = versionsDom.dataset.filepath;
+                        link.href = version.suffix_with_filename ? version.url + filepath : version.url;
+                        versionsDom.appendChild(link);
+                        if(version.version === versionsDom.dataset.majorVersion){
```
Comment (NolwennP) [inline src/js/versions.js:32]:
```suggestion
                        if(version.version === versionsDom.dataset.currentDocVersion){
```

File: src/partials/version-navigation.handlebars [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<div class="nav navbar-nav navbar-right">
+    <li class="active">
+        <a href="/pim/serenity/index.html">Serenity</a>
+    </li>
+    <li id="last-EE-version" data-filepath="{{filePath}}" data-major-version="{{majorVersion}}"></li>
```
Comment (NolwennP) [inline src/partials/version-navigation.handlebars:5]:
```suggestion
    <li id="last-EE-version" data-filepath="{{filePath}}" data-current-doc-version="{{majorVersion}}"></li>
```

**Pull request merged**